app/views/mailers/reservation_mailer/car_reservation_confirmation.html.erb

```diff
@@ -16,14 +16,16 @@
       <ul>
         <li>DATE: <%= "#{@start_time} - #{@end_time}" %></li>
         <li>VEHICLE #: <%= link_to @reservation.car.car_number, car_url(@reservation.car) %></li>
-        
         <li>SITE: <%= link_to @reservation.site.title, site_url(@reservation.site) %> </li>
         <li>PROGRAM: <%= link_to @reservation.program.title, program_url(@reservation.program) %></li>
-
-      You can email psych.transportation@umich.edu or call the office at: 734-764-2580.
+      </ul>
+    </p>
+    <p>
+      You can email psych.transportation@umich.edu or call the office at: <%= @contact_phone %>.
```

Comment:
Do the same variable handling for email like you are doing for phone number

Reply:
done here [00dd88a](https://github.com/lsa-mis/lsa_ride_share/pull/160/commits/00dd88a8a327f6d33106fb2e430a6f2ad70d236d)